xblock/django/request.py

```diff
@@ -21,7 +20,7 @@ def webob_to_django_response(webob_response):
     return django_response
 
 
-class HeaderDict(MutableMapping, collections.Iterator):  # pylint: disable=useless-object-inheritance
+class HeaderDict(MutableMapping):
```

Comment:
This class uses a `yield` in the `__iter__()` method, making that method a generator method - which is a fancy version of an iterator. So the class isn't technically an `Iterator` and never has been. An Iterator class would return `self` in the `__iter__()` method and implement `__next__()` using state within the instance.